src/components/Table/TableRowCell.react.js

```diff
@@ -0,0 +1,11 @@
+var TableRowCell = React.createClass({
```

Comment:
Not forgotten about, just haven't moved the rows out of the table component yet. I'll try and get that into this release.